src/components/Signup.jsx

```diff
@@ -1,99 +1,128 @@
 import React from "react";
-import ReactDOM from "react-dom";
 
-function Signup(){
-console.log("Sign up called");
-return (<div class="card">
-
-    <h5 class="card-header info-color white-text text-center py-4">
+function Signup() {
+  console.log("Sign up called");
+  return (
+    <div className="card">
+      <h5 className="card-header info-color white-text text-center py-4">
         <strong>Sign up</strong>
-    </h5>
```

Comment:
formatting file using the vscode extension mentioned before, converting <a href="" ... role="button"> to actual buttons